### ISSUE TITLE
[Concurrent] Breakpoint on MSE cannot work for multiple reasons (see details) 

### ISSUE THREAD

**Issue by ebousse** [gemoc/gemoc-studio#42]
In concurrent mode, in the "Stimuli manager" view, there is the possibility to add a breakpoint on an MSE. This break will never happen.

With the current code, this will never work for two reasons:
- This stores a breakpoint for the timesquare object "fr.inria.aoste.timesquare.ecl.feedback.feedback.impl.ModelSpecificEventImpl", and not for the corresponding GEMOC object "fr.inria.diverse.trace.commons.model.trace.MSE"
- The concurrent debugger only calls `shouldBreak` on the starting `Step`, and never on the `MSEOccurrence` that may trigger this step

To fix this problem, we must find the underlying MSE when creating breakpoints, and we must fix how breakpoints are checked in concurrent mode.

**Comment by dvojtise** [comment]
Right, the various refactoring of the trace MM has not been fully applied on the debug part of the concurrent mode. first point should be relatively easy to fix.

**Comment by ebousse** [comment]
After today's call to discuss many debugging-related points, we decided to have in the mid/long term different kinds of breakpoints on different kinds of elements (object, step, pair <object,step>, etc), but to focus for now on the case "pausing on an element".

After these changes, as soon as possible, we can add again the possibility to pause on an MSE, and therefore solve this issue.
